libnd4j/tests_cpu/layers_tests/DeclarableOpsTests10.cpp

```diff
@@ -2112,6 +2208,54 @@ TEST_F(DeclarableOpsTests10, ImageResizeNeighbor_Test1_1) {
     delete results;
 }
 
+TEST_F(DeclarableOpsTests10, ImageResizeNeighbor_Test1_1_1) {
+
+    NDArray input    = NDArrayFactory::create<float>('c', {1, 2, 3, 4});
+    //NDArray<float> paddings('c', {3,2}, {0,0, 0,1, 0,0});
+    //NDArray<float> expected('c', {2,4,4}, {1.,1.,1.,1.,1.,1.,1.,1.,1.,1.,1.,1.,0.,0.,0.,0.,1.,1.,1.,1.,1.,1.,1.,1.,1.,1.,1.,1.,0.,0.,0.,0.});
+    NDArray expected = NDArrayFactory::create<float>('c', {1, 4, 5, 4}, {
+            1.,  2.,  3.,  4.,
```

Comment:
Since you're creating float array, please add `f` literal, otherwise MSVC will fail building this.